src/containers/SubscriptionsPage/reducers.js

```diff
@@ -1,4 +1,4 @@
-import { fromJS } from 'immutable';
+import Immutable, { fromJS } from 'immutable';
```

Comment:
Better to `import { fromJS, List } from 'immutable'` as that will allow us to reduce the code size in the final build (webpack 2 will support dead code elimination).